client/client.go

```diff
@@ -45,6 +47,16 @@ func NewClient(token string) Client {
 	retClient.RetryWaitMin = 5 * time.Second
 	retClient.RetryWaitMax = 10 * time.Second
 
+	retClient.CheckRetry = func(ctx context.Context, resp *http.Response, err error) (bool, error) {
+		ok, e := retryablehttp.DefaultRetryPolicy(ctx, resp, err)
+		if !ok && resp.StatusCode == http.StatusUnauthorized {
```

Comment:
We better include a note about why we are doing this, retrying HTTP 401 normally doesn't make sense.

Reply:
Retrying on 401 did not help after all. (401 persists in subsequent queries as well).

Reply:
removed this